thirdparty/misc/clipper.cpp

```diff
@@ -101,8 +103,8 @@ namespace clipperlib {
 
   PolyPath& PolyPath::GetChild(unsigned index)
   {
-    if (index < 0 || index >= childs_.size())
-      throw ClipperException("invalid range in PolyPath::GetChild.");
+    // ERR_FAIL_INDEX_V(index, childs_.size(), PolyPath());
```

Comment:
Not sure how to return non-const reference here...

Reply:
Correct me if I'm wrong, but I'm don't think we're supposed to be touching thirdparty sources unless it's a bugfix or we're updating it to a new version -- any code related to Godot is supposed to go into the related `modules/` files, to my knowledge.

Reply:
I'm not sure either, it wouldn't compile otherwise for some other platforms with not exception-safe code.

Reply:
Seeing [thirdparty/README.md](https://github.com/godotengine/godot/blob/master/thirdparty/README.md) gives a hint that it can be done via denoting changes and/or generating an actual diff file against the original source.